Test.MegaCorpClash.Models/ChatCommandHandlers/TestHireCommandHandler.cs

```diff
@@ -24,57 +24,20 @@ public TestHireCommandHandler()
         });
     }
 
-    [Fact]
-    public void Test_NoCompany()
-    {
-        Dictionary<string, Company> companies = new();
-
-        var commandHandler =
-            new HireCommandHandler(_gameSettings, companies);
-
-        var gameCommand = GetGameCommand("!hire");
-
-        var chatMessageEvent =
-            Assert.Raises<ChatMessageEventArgs>(
-                h => commandHandler.OnChatMessagePublished += h,
-                h => commandHandler.OnChatMessagePublished -= h,
-                () => commandHandler.Execute(gameCommand));
-
-        Assert.NotNull(chatMessageEvent);
-        Assert.Equal(DEFAULT_CHATTER_DISPLAY_NAME, 
-            chatMessageEvent.Arguments.ChatterDisplayName);
-        Assert.Equal(Literals.YouDoNotHaveACompany, 
-            chatMessageEvent.Arguments.Message);
-    }
-
-    [Fact]
-    public void Test_NoParameters()
+    [Theory]
+    [InlineData("NoParameters", "")]
+    [InlineData("OneNumericParameter", "1")]
+    [InlineData("OneInvalidTextParameter", "asd")]
+    [InlineData("TwoTextParameters", "asd qwe")]
+    [InlineData("TwoValidButNoQty", "Sales Production")]
+    [InlineData("ThreeParameters", "1 2 3")]
+    [InlineData("InvalidJobType", "CEO 1")]
```

Comment:
name just used for readability with in test runner 
i.e if a case fails what case where we meant to be testing
![image](https://user-images.githubusercontent.com/4176258/189545983-1ec78ae0-c195-4016-8298-c228d18dc5d7.png)

Reply:
I can remove it if your not fussed